group_vars/all/vars.yml

```diff
@@ -58,6 +58,7 @@ version: '{{ python_app_version.stdout }}'
 logging_dir: '{{ install_root }}/logs'
 # logging file
 logging_path: '{{ logging_dir }}/{{ django_app }}.log'
+django_logging_path: '{{ logging_path }}'
```

Comment:
is the reason for having two of these so that we can also support RC apps that use `logging_path`?

Reply:
It's used in the `configure_logging` role, which I'm still using in the new shxco_qa playbook.

Would it make sense to create a logging setup task within the django role ? (although it's not django specific)